complete/src/main/java/com/example/messagingstompwebsocket/GreetingController.java

```diff
@@ -2,18 +2,26 @@
 
 import org.springframework.messaging.handler.annotation.MessageMapping;
 import org.springframework.messaging.handler.annotation.SendTo;
+import org.springframework.messaging.simp.annotation.SendToUser;
 import org.springframework.stereotype.Controller;
 import org.springframework.web.util.HtmlUtils;
 
 @Controller
 public class GreetingController {
 
 
-	@MessageMapping("/hello")
+	@MessageMapping("/hello/everyone")
 	@SendTo("/topic/greetings")
 	public Greeting greeting(HelloMessage message) throws Exception {
 		Thread.sleep(1000); // simulated delay
-		return new Greeting("Hello, " + HtmlUtils.htmlEscape(message.getName()) + "!");
+		return new Greeting("Hello everyone, I am " + HtmlUtils.htmlEscape(message.getName()) + "!");
 	}
 
+    @MessageMapping("/hello/user")
+    @SendToUser("/topic/greetings")
```

Comment:
I have a small question, which is how to specify which user to 'send back' after adding the SendToUser annotation. There is no specific destination specified here, nor is there a response ID or username field indicating it, @DLX4

Reply:
The user is the visitor to the web page rather than a certain account, so there's no need to specify.

Reply:
ok, i see, thanks.